### ISSUE TITLE
Initialize database error

### ISSUE THREAD

**Issue by SirZazu** [matecat/MateCat#768]
Hi,
I followed the official installation instruction and everything was fine till this step: https://www.matecat.com/advanced-manual-setup/#itd

unfortunately sending the database template to my MySQL instance ends up with error:
```ERROR 1364 (HY000) at line 1001: Field 'id_project' doesn't have a default value```

My version of MySQL:
```mysql  Ver 14.14 Distrib 5.7.17```
The distribution I am using:
```Linux ubuntu-artful 4.10.0-20-generic```

I have tried the `master` and the `dev` branch. The same issue on both. 
Any thoughts how can I finalise my installation?

**Comment by Ostico** [comment]
Hi @SimZal 

unfortunately, the instruction are correct but the SQL file you used are not. 
It is not up to date with the new changes in the schema.

Drop your new created schema and use this sql file to create a new one.

[matecat_schema.sql.zip](https://github.com/matecat/MateCat/files/984512/matecat_schema.sql.zip)

Let me know if worked.

**Comment by SirZazu** [comment]
@Ostico Thank you for your response. Unfortunately, the dump you have provided is giving me the same error as previously, but in a different line:
```
ERROR 1364 (HY000) at line 1093: Field 'id_project' doesn't have a default value
```
In order to pass this issue, I changed line `893` from:
```
`id_project` bigint(20) unsigned NOT NULL
```
to:
```
`id_project` bigint(20) unsigned DEFAULT NULL
```
I think it is secure because in line 1094 we have:
```
UPDATE `sequences` SET id_project = IFNULL( (SELECT MAX(id) + 1 FROM projects), 1);
```
But I am afraid, that this hack may have an impact on the system as I changed the field property and now this field can be null. 

Do you have any ideas about it? I would be happy to just load correct dump without modifying it.

**Comment by SirZazu** [comment]
@Ostico  Yes, your suggestion about modifying `/etc/mysql/my.cnf` fixed the problem and the dump you have attached in this issue works like a charm. Thank you very much.

**Comment by mzeidhassan** [comment]
I had hard time with this error. I have MySQL 5.7.21.

To make it work, I used the new schema file and I added the following to my.cnf file:
```

[mysqld]
sql-mode="NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"

```
restarted mysql server, then running command:

`mysql -u root -p < matecat.sql` didn't throw any errors. 

I thought I should update it, in case someone else is stuck.

I do hope you update the Advanced manual instructions guide as soon as possible. It is really outdated. 

Thanks,
Mohamed
